lua/acf/core/globals.lua

```diff
@@ -121,13 +121,11 @@ do -- ACF global vars
 	ACF.GunInaccuracyBias  = 2 -- Higher numbers make shots more likely to be inaccurate. Choose between 0.5 to 4. Default is 2 (unbiased).
 
 	-- Fuel
-	ACF.FuelRate           = 1 --multiplier for fuel usage, 1.0 is approx real world
+	ACF.FuelRate           = 1 -- Multiplier for fuel usage, 1.0 is approx real world
 	ACF.FuelFactor         = 1 -- Multiplier for ACF.FuelRate
```

Comment:
imo this should be set to the comp multiplier of 27.8

The rate of 27.8 was selected to influence larger vehicle builds as the realistic consumption rate is too small to realistically be a concern in AFV design

Reply:
Should be good now